### PR TITLE
Add ability to generate examples from icons.

### PR DESCRIPTION
**Pull request by JimiC**
I created this tool to help me preview how the new SVG icons will look in the editor and thought of sharing it with the rest.

**How it works:**
Read all about it in `README`.

### CODE REVIEW

**Comment by JimiC** [comment]
@robertohuertasm Please be so kind to review the code. I don't if my coding style follows yours, but I'm used to writing IIFE (so not to pollute the global namespace).

**Comment by robertohuertasm** [comment]
Hi @JimiC, what's the idea behind this PR? I'm not sure to get it 😞 
About coding style, being this a node module I wouldn't care about polluting the global namespace but it's ok for me if you like using IIFEs 😉

**Comment by JimiC** [comment]
As I wrote in README 

>In order to help you preview how the icon you are contributing will look in the editor, we are providing you with a tool that generates dummy folders or files of the provided icon. 

As for the coding style, I can switch it to match your style, no big deal for me. It's all about consistency, and I have an O.C.D. with that. So say the word and I'll do the switch.

**Comment by robertohuertasm** [comment]
I read the README but I didn't get the idea until now 😓 👍 
It seems a good idea to me 😉
Regarding coding style, I have no big issues with this as long as the code is readable and follows the eslint rules. But if you have such condition 😜 I would say I usually favour function declarations over function expressions and I'm not really into IIFEs unless really needed since working with modules provides such type of code encapsulation, but again, no sweat about this.

**Comment by JimiC** [comment]
@robertohuertasm OK. Hold your horses and don't merge it yet, until I switch the code.

**Comment by JimiC** [comment]
@robertohuertasm I'm done with the PR. Found some time at work to finish this.

File: src/build/supportedFolders.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ exports.extensions = {
     { icon: 'dist', extensions: ['dist', 'out', 'export', 'build'] },
     { icon: 'docs', extensions: ['docs'] },
     { icon: 'flow', extensions: ['flow'] },
-    { icon: 'git', extensions: ['git', 'github'], dot: true },
+    { icon: 'git', extensions: ['github', 'git'], dot: true },
```
Comment (JimiC) [inline src/build/supportedFolders.js]: I switched places because `.git` is getting ignored by `.gitignore`.

Comment (robertohuertasm) [inline src/build/supportedFolders.js]: What do you mean?

Comment (robertohuertasm) [inline src/build/supportedFolders.js]: yes, but what's the relationship between that and the order in the supported folders json?

Comment (robertohuertasm) [inline src/build/supportedFolders.js]: I already accepted the changes, it's just out of curiosity 😉

Comment (JimiC) [inline src/build/supportedFolders.js]: >  The generator only takes the first element from the supported extensions (no meaning to generate all extensions).

Comment (robertohuertasm) [inline src/build/supportedFolders.js]: Sorry, I missed that part. So it's related to the example generator. That makes sense.

Comment (JimiC) [inline src/build/supportedFolders.js]: My fault, I usually fail to fully explain in my writings.

**Pull request merged**